email/1.3/modules/ROOT/edits/email-send.adoc

```diff
@@ -0,0 +1,363 @@
++
+* *From address* +
+`user1@domain.com`
+* *To addresses* +
+`Edit inline`
++
+[start=13]
+. Click the plus sign (*+*) to add new emails.
+. Set the *Value* field to `user2@domain.com`.
+. Repeat the previous two actions and add another email as `user3@domain.com`.
+. Set the *Subject* field to `IMPORTANT!`.
+. Set the *Content* field to `"<h1>Hello this is an important message</h1>"` and the *ContentType* field to `text/html`. +
+If no body *Content* is specified, the content of the message payload is used by default. If that payload is not `text`, the operation fails with an `EMAIL:SEND` error.
+. Save and run the app.
+
+The following screenshot shows the *Send* operation configuration:
+
+.Email Send operation configuration
+image::email-send-configuration.png[Email Send operation configuration]
+
+=== XML for Sending Emails
+
+Paste this code into the Studio XML editor to quickly load the flow for this example into your Mule app:
+
+[source,xml,linenums]
+----
+<?xml version="1.0" encoding="UTF-8"?>
+
+<mule xmlns:tls="http://www.mulesoft.org/schema/mule/tls" xmlns:email="http://www.mulesoft.org/schema/mule/email"
+	xmlns="http://www.mulesoft.org/schema/mule/core"
+	xmlns:doc="http://www.mulesoft.org/schema/mule/documentation" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+http://www.mulesoft.org/schema/mule/email http://www.mulesoft.org/schema/mule/email/current/mule-email.xsd
+http://www.mulesoft.org/schema/mule/tls http://www.mulesoft.org/schema/mule/tls/current/mule-tls.xsd">
+	<email:smtp-config name="Email_SMTP" >
+		<email:smtps-connection host="pop.gmail.com" port="995" user="user1@domain.com" password="passwordvalue" >
+			<tls:context enabledProtocols="TLSv1.2,SSLv3" >
+				<tls:trust-store path="aTruststore.jks" password="changeit" />
+				<tls:key-store path="aKeystore" password="password" />
+			</tls:context>
+		</email:smtps-connection>
+	</email:smtp-config>
+	<flow name="emailconnectorFlow">
+		<scheduler>
+			<scheduling-strategy >
+				<fixed-frequency />
+			</scheduling-strategy>
+		</scheduler>
+		<email:send config-ref="Email_SMTP" fromAddress="user1@domain.com" subject="IMPORTANT!">
+			<email:to-addresses >
+				<email:to-address value="user3@domain.com" />
+        <email:to-address value="user2@domain.com" />
+			</email:to-addresses>
+			<email:body contentType="text/html" >
+				<email:content ><![CDATA["<h1>Hello this is an important message</h1>"]]></email:content>
+			</email:body>
+		</email:send>
+	</flow>
+</mule>
+----
+
+
+[[send-attachments]]
+== Send an Attachment
+
+The following example illustrates how to send emails and attachments over the SMTP server. Use DataWeave to handle the attachments. The flow reads a JSON file using the File Connector *Read* operation and then uses the Email Connector *Send* operation to send the content of the file as an attachment:
```

Comment:
"handle" > "process"?
I typically recommend not using "handle" unless within a context that mentions "handlers." Otherwise, the term can be temporarily misleading to nonnative English speakers.